using-valtimo/form-view-model/command-handling.md

```diff
@@ -0,0 +1,61 @@
+# Command handling
+
+A command is used to change the state of an application. Commands allow you to express clear intentions which simplify understanding and debugging of the application.
```

Comment:
Since the command handling moved to a new module, the documentation for this should not be at `form-view-model` anymore.